app/views/brexit_checker/show.html.erb

```diff
@@ -12,17 +12,21 @@
   end
 %>
 
+<% content_for :breadcrumbs do %>
 <div class="govuk-width-container">
```

Comment:
Same comment regarding indentation inside the `content_for` block.

Reply:
Thanks, have added that in the next commit, so that it's easier to see what has changed here